pkgs/tools/graphics/zbar/default.nix

```diff
@@ -68,7 +68,7 @@ stdenv.mkDerivation rec {
   dontWrapGApps = true;
 
   postFixup = lib.optionalString enableVideo ''
-    wrapProgram "$out/bin/zbarcam-gtk" "''${gappsWrapperArgs[@]}"
+    wrapGApp "$out/bin/zbarcam-gtk"
```

Comment:
wrapGAppsHook isn't in nativeBuildInputs

Reply:
Bah, missed that when rebasing #68389